package.json

```diff
@@ -53,13 +58,17 @@
     "less": "^4.1.3",
     "less-loader": "^11.1.0",
     "playwright": "^1.32.2",
-    "powerbi-visuals-api": "^5.3.0",
+    "powerbi-visuals-api": "~5.3.0",
     "powerbi-visuals-tools": "^4.2.1",
     "powerbi-visuals-utils-testutils": "^6.0.1",
     "style-loader": "^3.3.2",
+    "stylelint": "^14.0.0",
     "ts-node": "^10.9.1",
     "typescript": "^5.0.3",
-    "webpack": "^5.76.1"
+    "webpack": "^5.76.1",
+    "jquery": "3.5.1",
```

Comment:
please remove jquery and jamins-jquery form your PRs

Reply:
resolved